src/cli/Remove.cpp

```diff
@@ -35,7 +35,8 @@ int Remove::executeWithDatabase(QSharedPointer<Database> database, QSharedPointe
     auto& out = parser->isSet(Command::QuietOption) ? Utils::DEVNULL : Utils::STDOUT;
     auto& err = Utils::STDERR;
 
-    auto& entryPath = parser->positionalArguments().at(1);
+    auto args = parser->positionalArguments();
+    auto& entryPath = args.at(1);
```

Comment:
Even easier, just remove the & after auto in the original line. I don't think this is actually an error though, seems like the compiler is getting too sensitive.

Reply:
Thank for suggestion, fixed. I had the same idea but checked the code and found some examples with a separate variable.